godtools/push/action.yaml

```diff
@@ -26,6 +26,10 @@ inputs:
     required: false
     default: "false"
     description: Push `image:develop` from branch `develop` to the registry
+  skip-signing:
+    required: false
+    default: "false"
```

Comment:
@bkondek @jaceklubzinski I will change this one to `true` and merge this action already - that way we could introduce proper godtools config update, test out signing in controlled way and change this to `false` after that.